zen/screen-renderer.c

```diff
@@ -203,12 +206,12 @@ zn_screen_renderer_render_decoration(struct zn_screen *screen,
 
   {
     pixman_region32_t region;
-    struct wlr_fbox window_fbox;
+    struct wlr_fbox surface_fbox;
     struct wlr_box transformed_window_box;
 
-    zn_view_get_window_fbox(view, &window_fbox);
+    zn_view_get_surface_fbox(view, &surface_fbox);
```

Comment:
The name of "surface" and "window" is mixing (`surface_fbox` and `transformed_window_box`), and as you know, it's confusing part. How about using the name "window" and add comment about it?

Changing function name seems good, too.
`zn_screen_renderer_render_decoration` -> `zn_screen_renderer_render_server_decoration`

```suggestion
    zn_view_get_surface_fbox(view, &window_fbox); // Using server-side decoration, window_fbox and surface_fbox is the same
```